dawn/renderer/components/UpdateBox.js

```diff
@@ -35,12 +37,10 @@ class UpdateBox extends React.Component {
     const update = pathToName(this.state.updateFilepath);
     if (!update) {
       this.setState({ isUploading: false });
-      dialog.showMessageBox({
-        type: 'warning',
-        buttons: ['Close'],
-        title: 'File Issue',
-        message: 'Update File Bad\n',
-      });
+      this.props.onAlertAdd(
+        'File Issue',
+        'Update File Bad',
```

Comment:
Could we give this a more descriptive title/message, like "Issue with file upload" and "There was an issue uploading the current file."?